src/Symfony/Component/HttpClient/Response/CommonResponseTrait.php

```diff
@@ -54,6 +54,8 @@ public function getContent(bool $throw = true): string
             foreach (self::stream([$this]) as $chunk) {
                 if (!$chunk->isLast()) {
                     $content .= $chunk->getContent();
+                } elseif (null === $content) {
+                    $content = '';
```

Comment:
this cannot be correct (and the CI agrees :) )

Reply:
hmm, ok, I understand... the issue in not in AsyncResponse, but in no-buffer response. Here is a reproduct case:


```php
$client = HttpClient::create();
$resp = $client->request('GET', 'https://httpbin.org/status/429', ['buffer' => false]);
$resp->getContent(false);
```

Issue is, in that case, we have 2 chunks: `FirstChunk` and `LastChunk` (without data).
`FirstChunk` is swallowed by `CommonResponseTrait::initialize` letting only the `LastChunk` that did not let time to `$content` to get data.

I need helps to fix it here.

Reply:
See https://github.com/symfony/symfony/pull/38388